draft-ietf-rats-architecture.md

```diff
@@ -576,33 +648,33 @@ plans to support in the TEEP architecture {{?I-D.ietf-teep-architecture}}.
 # Trust Model {#trustmodel}
 
 The scope of this document is scenarios for which a Relying Party
-trusts a Verifier that can evaluate the trustworthiness of
+trusts a Verifier that can appraise the trustworthiness of
 information about an Attester.  Such trust might come by the Relying
 Party trusting the Verifier (or its public key) directly, or might
 come by trusting an entity (e.g., a Certificate Authority) that is
 in the Verifier's certificate chain.  The Relying Party
 might implicitly trust a Verifier (such as in the Verifying Relying
 Party combination).  Or, for a stronger level of security, the
 Relying Party might require that the Verifier itself provide
-information about itself that the Relying Party can use to evaluate
+information about itself that the Relying Party can use to assess
 the trustworthiness of the Verifier before accepting its Attestation Results.
 
 The Endorser and Verifier Owner may need to trust the Verifier
 before giving the Endorsement and Appraisal Policy to it.
 Such trust can also be established directly or indirectly,
 implicitly or explicitly. One explicit way to establish such trust
-may be the Verifier first acts as an Attester and attests to the
-Endorser and/or Verifier Owner as the Relying Parties.
+may be the Verifier first acts as an Attester and creates Evidence about itself to be consumed by the
```

Comment:
I prefer here unchanged. Evidence is consumed by a Verifier. Here wants to say the Verifier can act as an Attester to be considered trustworthy or not by the Endorse and/or Verifier Owner as the Relying Parties.